content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Configuring Gateway Network Topology (Development)
+description: How to configure gateway network topology (Development).
+weight: 60
+keywords: [traffic-management,ingress,gateway]
+---
+
+## (Development) Configuring network topologies
```

Comment:
Why it is tagged as Development?  would be good to explain to readers

Reply:
Yeah, I will add a line. Basically, we will move the API around as we haven't settled on the correct UX.

Reply:
I added this as "Development" is a feature stage and "Experimental" is not. I can update this to explain what Development means.

Reply:
Hopefully this makes more sense now as I've linked to the feature stages found in istio/community.